0243-tca-tour-pt1/Counter/Counter/ContentView.swift

```diff
@@ -11,7 +13,11 @@ extension NumberFactClient: DependencyKey {
     )
     return String(decoding: data, as: UTF8.self)
   }
+//  static let previewValue: NumberFactClient = Self { number in
+//    throw "Failed to fetch fact"
+//  }
```

Comment:
How can I inject this behavior directly into the preview? I want ONE preview to always fail, but the other one should work fine.